vuepress/docs/next/tutorials/devops/external-id-management.md

```diff
@@ -2,75 +2,83 @@
 
 ## Purpose
 
-This tutorial enables the Entando administrator to leverage an existing
-Keycloak instance.
+This tutorial details how Entando can be connected to an existing Keycloak instance.
 
-## Requirements
+## Prerequisites
 
 -   A Keycloak instance
-
--   A realm named "entando" on that instance
-
--   An admin user for the "entando" realm
+-   A realm named "entando" in that instance
+-   Admin user credentials for the "entando" realm
```

Comment:
These are the credentials (username/password) for the service account that has the correct level of admin permissions